.github/workflows/update-man-pages.yml

```diff
@@ -0,0 +1,39 @@
+name: Update man-pages workflow
+on:
+  schedule:
+    - cron: "42 3 1/15 * *"   # Trigger every 15 days at 03:42
+    #- cron: "0,5,10,15,20,25,30,35,40,45,50,55 * * * *"  # For testing
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-man-pages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository code
+        uses: actions/checkout@v4
+
+      - name: Update man-pages
+        run: |
+          set -euo pipefail
+
+          rm ./documentation/manpages/sdk/*
+
+          ./documentation/manpages/tool/run_docker.sh
+
+          if [[ -n $(git status -s) ]]; then
+            git config user.name 'TODO'
+            git config user.email 'TODO@example.com'
```

Comment:
Are there any standard/recommended name/email combinations I can use?

Reply:
We have a precedence:
```
eng/common/core-templates/steps/source-build.yml:        git config user.email dn-bot@microsoft.com
eng/common/core-templates/steps/source-build.yml-        git config user.name dn-bot
--
eng/pipelines/templates/steps/vmr-pull-updates.yml-    git config --global user.name "dotnet-maestro[bot]"
eng/pipelines/templates/steps/vmr-pull-updates.yml:    git config --global user.email "dotnet-maestro[bot]@users.noreply.github.com"
```

I think the idea is to run it automatically so we don't forget to update (we frequently forget to update the manpages).

Reply:
I wouldn't want to re-use those names without checking with the arcade and/or MS source-build teams first - we (SDK) don't control those identities and I wouldn't want to muddy contributions.

Reply:
cc @dotnet/product-construction. Could we tie the triggering to the release cadence? i.e. add to the list of prerequisites of release (both previews and GA). Job only took 42s to complete https://github.com/omajid/dotnet-sdk/actions/runs/14769460163.